ksql-engine/src/main/java/io/confluent/ksql/util/ExpressionTypeManager.java

```diff
@@ -198,10 +196,11 @@ protected Expression visitFunctionCall(final FunctionCall node,
       functionArguments.beginFunction();
```

Comment:
If we're not using functionArguments to track the call stack then can we just clean it up? Otherwise all its doing is being a list.

Reply:
Removed it.

Reply:
Removed it from this part.

Reply:
For my understanding, the stack is being replaced by multiple instances of `ExpressionTypeManager` now?

Reply:
Are you going to remove the `FunctionArguments`? Seems it is not needed with this change, i.e., as @rodesai suggested